spec/requests/v1/songs_request_spec.rb

```diff
@@ -13,6 +13,34 @@
       end
     end
 
+    context 'when there are more than 10 songs' do
+      context 'when no per_page param is specified' do
+        it 'returns JSON for 10 songs' do
+          create_list(:song, 100)
+
+          get(songs_url, {}, accept_headers)
+
+          expect(response).to have_http_status :ok
+          expect(response).to match_response_schema :songs
+          expect(response.headers['Total']).to eq '100'
+          expect(response.headers['Per-Page']).to eq '10'
+        end
+      end
+
+      context 'when a per_page param is specified' do
+        it 'returns JSON for the number of songs requested' do
+          create_list(:song, 100)
+
+          get(songs_url, { per_page: 5 }, accept_headers)
+
+          expect(response).to have_http_status :ok
+          expect(response).to match_response_schema :songs
+          expect(response.headers['Total']).to eq '100'
```

Comment:
So, what if we wanted to make a nice helper in https://github.com/dgrosenblatt/spotify-apprentice-project-server/blob/master/spec/support/helpers/requests.rb to make it easier to check the `total` header or the `per-page`  in https://github.com/dgrosenblatt/spotify-apprentice-project-server/blob/master/spec/support/helpers/requests.rb?

So, if I had:

```
total_songs = 100
per_page = 5

create_list(:song, total_songs)

get(songs_url, { per_page: per_page }, accept_headers)

expect(response).to have_http_status :ok
expect(response).to match_response_schema :songs

expect(response_total).to eq total_songs
expect(response_per_page).to eq per_page
```